buildpipeline/DotNet-CoreClr-Trusted-Mac.json

```diff
@@ -31,7 +31,7 @@
       },
       "inputs": {
         "filename": "$(Agent.BuildDirectory)/s/sync.sh",
-        "arguments": "",
+        "arguments": "-p -- /p:DotNetRestoreSources=$(PB_RestoreSource)",
```

Comment:
I suspect we are going to need this on more then just the sync command. Are you sure we don't do any restoring on other commands?